openfisca_us/parameters/benefit/SNAP/amounts/household_size.yaml

```diff
@@ -0,0 +1,1977 @@
+description:  Maximum Monthly SNAP Benefit by State and Household Size
```

Comment:
Per https://github.com/UBICenter/openfisca-us/issues/48#issuecomment-903856922 let's name this file `benefit/SNAP/amount.yaml`